### ISSUE TITLE
Support for Becker&Hickl FIFO files generated by SPC-1xx modules

### ISSUE THREAD

**Issue by lampo808** [Photon-HDF5/phconvert#16]
Currently the conversion of .spc files generated by Becker&Hickl boards is supported only for files generated by a SPC-630 board.
I plan to modify the bhreader module to enable the import of files generated by SPC-1xx modules in the near future.

## How?

My idea is to add an optional argument to `bhreader.load_spc` to specify the module that generated the file:
`def load_spc(fname):` --> `def load_spc(fname, spc_model='SPC-630'):`
so that the function can take care of different file formats.

In the future we could implement an additional function that reads both the .spc and .set files and automatically read the time-stamped data based on the identification of the board contained in the .set file (in a field I haven't identified yet).

**Comment by tritemio** [comment]
Sounds a good plan, thanks!

**Comment by tritemio** [comment]
@lampo808 for the identification of the files you can look into the `Identification` group in the SET file. The `Contents` field has information about the board model, at least for SPC-630 files.

**Comment by tritemio** [comment]
Specifically, in the case of SPC-630 the field contains:

```
Setup file made by system at the end of FIFO measurement with module SPC-630 (Ser.No. 3A0067)
```

**Comment by lampo808** [comment]
Great, thanks!

**Comment by tritemio** [comment]
Addressed by #18. Closing.
